primitive-types/tests/scale_info.rs

```diff
@@ -9,7 +9,7 @@
 //! Tests for scale-info feature of primitive-types.
 
 use primitive_types::{H256, U256};
-use scale_info_crate::{build::Fields, Path, Type, TypeInfo};
+use scale_info::{build::Fields, Path, Type, TypeInfo};
```

Comment:
I believe this also need to be reverted
```suggestion
use scale_info_crate::{build::Fields, Path, Type, TypeInfo};
```